storage/operation/collections.go

```diff
@@ -0,0 +1,47 @@
+package operation
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage"
+)
+
+// NOTE: These insert light collections, which only contain references
+// to the constituent transactions. They do not modify transactions contained
+// by the collections.
+
+func InsertCollection(w storage.Writer, collection *flow.LightCollection) error {
+	return UpsertByKey(w, MakePrefix(codeCollection, collection.ID()), collection)
+}
+
+func RetrieveCollection(r storage.Reader, collID flow.Identifier, collection *flow.LightCollection) error {
+	return RetrieveByKey(r, MakePrefix(codeCollection, collID), collection)
+}
+
+func RemoveCollection(w storage.Writer, collID flow.Identifier) error {
+	return RemoveByKey(w, MakePrefix(codeCollection, collID))
+}
+
+// IndexCollectionPayload indexes the transactions within the collection payload
+// of a cluster block.
```

Comment:
is this specific to collection cluster logic, or is this just indexing by blockID?

I'm wondering if we're overloading the `codeIndexCollection` to mean different things on ANs/ENs vs LNs